recipes/dlib_for_biorbd/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "dlib_for_biorbd" %}
+{% set version = "19.10.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pyomeca/dlib/archive/{{ version }}.tar.gz
+  sha256: 57326f799022dbc340ebb27f4e1f1471da6eb949b42fcac008dbee068f088df6
+  
+build:
+  number: 0
+  skip: True  # [win]
+  features:
+    - vc14  # [win]
+  rpaths:
+    - lib/
+    - lib/dlib/
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}  # [linux]
+    - {{ compiler('gcc') }}  # [mac]
```

Comment:
Hmmm this doesn't seem like the right way to select compiler...on my phone so I can't investigate

Reply:
For some reason, compiler('cxx') doesn't work with osx.. It returns some error about the fact that clang is not created. On internet, some guys ran into similar problems and it was due to incompatibility between CMake and Clang.

They were proposing solutions, but all involved changing the CMakeLists.txt file, which I prefer not to (since obviously I am not the main developer for dlib!). 

So, the only work around was to actually compile using gcc instead

Reply:
I think this is causing the host's compiler to be used instead of a conda-forge one.

The easiest solution is to add a patch to increase the `cmake_minimum_required` to 3.2. Alternatively I think there is a flag you can pass to set the policies correctly but I don't remember exactly how.